moveit_ros/planning/robot_model_loader/src/robot_model_loader.cpp

```diff
@@ -131,55 +131,69 @@ void RobotModelLoader::configure(const Options& opt)
         std::string param_name;
         try
         {
-          param_name = prefix + "max_position";
-          double max_position;
-          if (node_->get_parameter(param_name, max_position))
+          param_name = prefix + "has_position_limits";
+          bool has_position_limits = node_->declare_parameter(param_name, false);
+          if (node_->get_parameter(param_name, has_position_limits))
```

Comment:
This will always return true since we're declaring it with a default value, also we need to make sure that a parameter wasn't declared before otherwise, it will throw an exception

Same for the other parameters